reporter/src/main/kotlin/LicenseTextProvider.kt

```diff
@@ -0,0 +1,9 @@
+package com.here.ort.reporter.reporters
+
+import com.here.ort.spdx.getLicenseText
+
+class LicenseTextProvider {
```

Comment:
Shouldn't this be / implement an interface, similar to `ResolutionProvider`?

Reply:
done.